src/renderer/features/governance/ReferendumList/lib/referendum-list-utils.ts

```diff
@@ -1,7 +1,76 @@
+import { BN } from '@polkadot/util';
+import orderBy from 'lodash/orderBy';
+
+import { IconNames } from '@shared/ui/Icon/data';
+import {
+  ReferendumId,
+  OngoingReferendum,
+  CompletedReferendum,
+  TrackId,
+  Tally,
+  Voting,
+  VotingType,
+  CastingVoting,
+  type Address,
+} from '@shared/core';
+
 export const referendumListUtils = {
-  test,
+  getSortedOngoing,
+  getSortedCompleted,
+  getVoteFractions,
+  getTrackInfo,
+  isReferendumVoted,
 };
 
-function test() {
-  console.log(1);
+function getSortedOngoing(referendums: Map<ReferendumId, OngoingReferendum>): Map<ReferendumId, OngoingReferendum> {
+  return new Map(orderBy(Array.from(referendums), ([index]) => parseInt(index), 'desc'));
+}
+
+function getSortedCompleted(
+  referendums: Map<ReferendumId, CompletedReferendum>,
+): Map<ReferendumId, CompletedReferendum> {
+  return new Map(orderBy(Array.from(referendums), ([index]) => parseInt(index), 'desc'));
+}
```

Comment:
Are there any differences but type? The functions look the same maybe combine them in one or will there be more in these functions?

Reply:
I'll leave a `todo` because sorting should be updated